pkg/flavor/kubernetes/nfs.go

```diff
@@ -359,6 +368,7 @@ func (flavor *Flavor) HandleNFSNodePublish(chapiDriver chapi.Driver, req *csi.No
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	chapiDriver := &chapi.LinuxDriver{}
```

Comment:
What if this is running on Windows?  Should we check the OS type?

Not that we support that now.  I'm just asking.

Reply:
sure, if we use the same reference created in driver, that should be handled using getDriver() call.

Reply:
Exactly.